lib/signaling/v2/peerconnection.js

```diff
@@ -1000,4 +1002,68 @@ function filterOutMediaStreamIds(sdp) {
   return sdp.replace(/a=msid:[^ ]+ /g, 'a=msid:- ');
 }
 
+/**
+ * @param {string} sdp
+ * @param {string} mid
+ * @returns {?string} direction
+ */
+function getTransceiverDirection(sdp, mid) {
+  const section = sdp.split('\r\nm=').find(section => section.match(`a=mid:${mid}`));
+  if (!section) {
+    return null;
+  } else if (section.match(/a=sendrecv/)) {
```

Comment:
@markandrus This can be:
```js
const matches = section.match(/a=(sendrecv|sendonly|recvonly|inactive)/);
if (!matches) {
  return null;
}
return matches[1];
```